youxia-q2seqware/src/main/java/io/cloudbindle/youxia/q2seqware/RabbitMQListener.java

```diff
@@ -0,0 +1,163 @@
+package io.cloudbindle.youxia.q2seqware;
+
+import io.cloudbindle.youxia.util.Log;
+
+import java.io.IOException;
+
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+import com.rabbitmq.client.ConsumerCancelledException;
+import com.rabbitmq.client.ShutdownSignalException;
+
+public class RabbitMQListener implements QueueListener {
+
+    private String hostName;
+    private String queueName;
+    private boolean autoAck;
+    private boolean durableQueue;
+    private String pathToINIs = "/tmp/inis4seqware/";
```

Comment:
Currently, the path the INI files is in the q2seqware.ini file (the path in tmp is a default), in case someone wants to configure the path somewhere else.

I can migrate the config settings from this file to ~/.youxia/config but will probably get this on a later commit.